src/metabase/lib/core.cljc

```diff
@@ -437,16 +441,23 @@
   clause?
   clause-of-type?
   fresh-uuids
+  mbql-stage?
   native-stage?
   normalized-query-type
+  normalized-mbql-version
```

Comment:
Add a function to easily check if a query map is `:mbql-version/legacy` or `:mbql-version/mbql5`